Enhancing troubleshoot section with Windows Operational Readiness (#45612)
kubernetes/website: content/en/docs/concepts/windows/intro.md

The Kubernetes [cluster API](https://cluster-api.sigs.k8s.io/) project also provides means to automate deployment of Windows nodes.

### Validating the Windows cluster operability

> The Windows Operational Readiness is a sub-project and specification of the SIG Windows,
> which introduces a structured test suite. This suite is split into two sets of tests,

Reviewer: ```suggestion
The Kubernetes project provides a _Windows Operational Readiness_ specification,
accompanied by structured test suite. This suite is split into two sets of tests,
```
